views/error.hbs

```diff
@@ -1,23 +1,37 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <title>Error Page - IDEA Hacks</title>
     <meta charset="UTF-8">
 
```

Comment:
You might have noticed that when the screen gets small, everything gets really tiny instead of wrapping the text. This is because whoever set up this page forget a second `<meta>` tag: `<meta name="viewport" content="width=device-width, initial-scale=1.0">`. This is a new standard that lets webpages play nicely with phones. You can just add this line directly under the first `<meta>` tag